incubator/ibm-blockchain-chaincode/README.md

```diff
@@ -0,0 +1,60 @@
+# ibm-blockchain-chaincode
+
+Use [ibm-blockchain](https://ibm-blockchain.github.io) to develop in a cloud sandbox using the IBM Blockchain Platform.
+
+## TL;DR;
+
+```console
+$ helm install stable/ibm-blockchain-chaincode
+```
+
+## Introduction
+
+This chart bootstraps a [ibm-blockchain](https://ibm-blockchain.github.io) deployment on a [Kubernetes](http://kubernetes.io) cluster using the [Helm](https://helm.sh) package manager.
+
+## Prerequisites
+
+- Kubernetes 1.4+ with Beta APIs enabled
```

Comment:
We officially support the current and previous minor versions, i. e. 1.7+

Reply:
fixed